index.bs

```diff
@@ -688,13 +685,25 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 			1. For each (|key|, |value|) in |items|:
 
+				1. Let |representation| be a new [=representation=].
+
+				1. Let |isCustom| be |false|.
+				
+				1. If |key| has "web " ("web" followed by U+0020 SPACE) prefix, then 
```

Comment:
Is there a "starts with" definition for strings we can reference here?

Reply:
https://infra.spec.whatwg.org/#string-starts-with

Reply:
Will make this change, thanks!

Reply:
https://github.com/w3c/clipboard-apis/pull/195